src/CoreWCF.UnixDomainSocket/src/CoreWCF/UnixDomainSocketBinding.cs

```diff
@@ -0,0 +1,144 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.ComponentModel;
+using System.Xml;
+using CoreWCF.Channels;
+
+namespace CoreWCF
+{
+    public class UnixDomainSocketBinding : Binding
+    {
+        // private BindingElements
+        private UnixDomainSocketTransportBindingElement _transport;
+        private BinaryMessageEncodingBindingElement _encoding;
+        private UnixDomainSocketSecurity _security = new UnixDomainSocketSecurity();
+
+        public UnixDomainSocketBinding() { Initialize(); }
+        public UnixDomainSocketBinding(SecurityMode securityMode)
+            : this()
+        {
+            _security.Mode = securityMode;
+        }
+
+        public TransferMode TransferMode
+        {
+            get { return _transport.TransferMode; }
+            set { _transport.TransferMode = value; }
+        }
+
+        public HostNameComparisonMode HostNameComparisonMode
+        {
+            get { return _transport.HostNameComparisonMode; }
+            set { _transport.HostNameComparisonMode = value; }
+        }
+
+        [DefaultValue(TransportDefaults.MaxBufferPoolSize)]
+        public long MaxBufferPoolSize
+        {
+            get { return _transport.MaxBufferPoolSize; }
+            set
+            {
+                _transport.MaxBufferPoolSize = value;
+            }
+        }
+
+        public int MaxBufferSize
+        {
+            get { return _transport.MaxBufferSize; }
+            set { _transport.MaxBufferSize = value; }
+        }
+
+        public int MaxConnections
+        {
+            get { return _transport.MaxPendingConnections; }
+            set
+            {
+                _transport.MaxPendingConnections = value;
+            }
+        }
+
+        public int ListenBacklog
+        {
```

Comment:
I don't think we pass this value along to Kestrel. I don't think we even can modify this value. Unless this is wired up to modifying some config value for Kestrel, remove it and the corresponding property in the transport binding element.